### PR TITLE
Drop EOL kinetic support

### LINKED ISSUE #15: Build failed
Hi @orzechow

I got notified that the build failed, maybe you want to watch this repo from now on 😅 

https://github.com/coincar-sim/lanelet_rviz_plugin_ros/runs/2679110720

### PR DESCRIPTION
**Pull request by orzechow**
Closes #15

### CODE REVIEW

**Comment by orzechow** [comment]
Due to unstable key servers these builds failed a couple of times, so I changed to use gpg keys directly in `travis.sh`:
https://github.com/coincar-sim/mrt_cmake_modules_ci/pull/7

This should fix the build of this PR.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This config file for Travis CI utilizes https://github.com/coincar-sim/mrt_cmake_modules_ci/ package.
 sudo: required
-dist: xenial
+dist: bionic
```
Comment (m-naumann) [inline .travis.yml:3]: Is travis' bionic stable these days? It used to be a good idea to always wait for quite some time. That's why I only "recently" switched from trusty (https://github.com/coincar-sim/mrt_cmake_modules_ci/commit/08117b35f99bd530858ea55074f05bbfbdf30af9)

**Review by m-naumann — COMMENTED** [review]
That's fine if it reached EOL. @einsteinguang @eduardo-molinos could you check if this still in use in other coincar-sim repos? Thanks :)

File: .travis.yml (hunk 1 above)
Comment (orzechow) [reply .travis.yml:3]: Did you have issues with Travis bionic? I'd still assume that bionic should be better than EOL xenial ;)

**Comment by orzechow** [comment]
See, that was the problem ;)

**Pull request merged**